# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmGenerator.cpp
# Commit: 538920d2d777
# Full Hash: 538920d2d77751743a51f10cf730f1ce65b5d0ee
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2020-08-18 09:24:52
# Regressor Bug: 1561521
# File Overlap Count: 2
# Description:
#   Bug 1561521 - Rename TypeCode/ValType/RefType::Any to Extern. r=lth
#   
#   This commit renames the TypeCode/ValType/RefType occurrences of 'any' to 'extern'
#   to match what they actually represent. All public API facing occurrences have
#   already been switched to 'externref' so this should only be aligning the
# ==============================================================================

diff -r 3d3e3d124767 -r 538920d2d777 js/src/wasm/WasmGenerator.cpp
--- a/js/src/wasm/WasmGenerator.cpp	Tue Aug 18 00:03:29 2020 +0000
+++ b/js/src/wasm/WasmGenerator.cpp	Tue Aug 18 00:04:37 2020 +0000
@@ -383,7 +383,7 @@
   for (const ElemSegment* seg : env_->elemSegments) {
     // For now, the segments always carry function indices regardless of the
     // segment's declared element type; this works because the only legal
-    // element types are funcref and anyref and the only legal values are
+    // element types are funcref and externref and the only legal values are
     // functions and null.  We always add functions in segments as exported
     // functions, regardless of the segment's type.  In the future, if we make
     // the representation of AnyRef segments different, we will have to consider